src/analytics/Properties.tsx

```diff
@@ -491,8 +491,8 @@ interface EscrowEventsProperties {
 interface SendEventsProperties {
   [SendEvents.send_scan]: undefined
   [SendEvents.send_select_recipient]: {
-    // TODO: decide what recipient info to collect, now that RecipientKind doesn't exist
     usedSearchBar: boolean
+    recipientType?: RecipientType
```

Comment:
do you know what the cases are where recipientType is undefined? it would be helpful to document in a code comment here since I think this is where people look to understand the analytics data